app/components/file_attachment_list/file_attachment_document/file_attachment_document.js

```diff
@@ -146,7 +146,7 @@ export default class FileAttachmentDocument extends PureComponent {
                 this.setState({downloading: true});
                 this.downloadTask = RNFetchBlob.config(options).fetch('GET', getFileUrl(data.id));
                 this.downloadTask.progress((received, total) => {
-                    const progress = (received / total) * 100;
+                    const progress = Math.round((received / total) * 100);
```

Comment:
This should fix the recently-reported bug in the progress circle. Conversation [here](https://community.mattermost.com/private-core/pl/mo91r1bdqfnozf1ocd8ng4n7yw) .

Reply:
good call, no bug was found.. but while testing this PR I came across the issue and added the fix :)